CoffeeChat/Controller/EditInterestViewController.swift

```diff
@@ -0,0 +1,195 @@
+//
+//  EditInterestVireController.swift
+//  CoffeeChat
+//
+//  Created by Phillip OReggio on 3/11/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+
+// MARK: - UIViewController
+class EditInterestViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let interestsTableView = UITableView(frame: .zero, style: .grouped)
+
+    // MARK: - Collection View Sections
+    private enum Section {
+        case yourInterests([Interest])
+        case moreInterests([Interest])
+    }
+
+    // MARK: - Data Vars
+    // TODO replace with actual data
+    private var yourInterests: [Interest] = [
+        Interest(name: "Art", categories: "lorem, lorem, lorem, lorem, lorem", image: "art"),
+        Interest(name: "Business", categories: "lorem, lorem, lorem, lorem, lorem", image: "business"),
+        Interest(name: "Dance", categories: "lorem, lorem, lorem, lorem, lorem", image: "dance")
+    ]
+    private var moreInterests: [Interest] = [
+        Interest(name: "Design", categories: "lorem, lorem, lorem, lorem, lorem", image: "design"),
+        Interest(name: "Fashion", categories: "lorem, lorem, lorem, lorem, lorem", image: "fashion"),
+        Interest(name: "Fitness", categories: "lorem, lorem, lorem, lorem, lorem", image: "fitness"),
+        Interest(name: "Food", categories: "lorem, lorem, lorem, lorem, lorem", image: "food"),
+        Interest(name: "Humanities", categories: "lorem, lorem, lorem, lorem, lorem", image: "humanities"),
+        Interest(name: "Music", categories: "lorem, lorem, lorem, lorem, lorem", image: "music"),
+        Interest(name: "Photography", categories: "lorem, lorem, lorem, lorem, lorem", image: "photography"),
+        Interest(name: "Reading", categories: "lorem, lorem, lorem, lorem, lorem", image: "reading"),
+        Interest(name: "Sustainability", categories: "lorem, lorem, lorem, lorem, lorem", image: "sustainability"),
+        Interest(name: "Technology", categories: "lorem, lorem, lorem, lorem, lorem", image: "tech"),
+        Interest(name: "Travel", categories: "lorem, lorem, lorem, lorem, lorem", image: "travel"),
+        Interest(name: "TV & Film", categories: "lorem, lorem, lorem, lorem, lorem", image: "tvfilm")
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .backgroundLightGreen
+
```

Comment:
Just wanted to make a comment about adding the navigation bar? I think you can just copy from my PR?